src/common/utils/setup-helpers.js

```diff
@@ -104,6 +104,22 @@ function setStorageTextStyle(player: Player): void {
   }
 }
 
+/**
+ * Call to getCapabilities on the first UI_CLICKED event
+ * @param {Player} player - The Kaltura player.
+ * @returns {void}
+ */
+function attachToFirstClick(player: Player): void {
+  if (isIos()) {
+    const onUIClicked = () => {
+      player.removeEventListener(player.Event.UI.UI_CLICKED, onUIClicked);
+      // Trigger to load the auto play testing video tag
+      getCapabilities();
+    };
+    player.addEventListener(player.Event.UI.UI_CLICKED, onUIClicked);
```

Comment:
This change assumes that by the first `UI_CLICKED` the main video tag is always can auto play.
But this assumption is **wrong!!!**. 
Consider a case that the first entry request is failed, and the player is in an error state.
By click on the error overlay (for retry for example) we get `UI_CLICKED` but the main video tag is still **cannot** auto play, as it has never played.